docs/pages/web/popovereducational.js

```diff
@@ -0,0 +1,628 @@
+            anchor={anchorRefB.current}
+            onDismiss={() => {}}
+            message="Explore your recent searches in more details"
+          />
+        )}
+      </Flex>
+    </Box>
+)}
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Position PopoverEducational appropriately on the screen. Make sure the arrow points directly to the element it is referencing."
+            defaultCode={`
+function Example() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+
+  React.useEffect(() => {
+    setOpen(true)
+  }, []);
+
+  return (
+    <React.Fragment>
+      <TapArea fullWidth={false} rounding={3}>
+        <Box borderStyle="shadow" padding={3} rounding={3} width={400}>
+          <Flex direction="column" gap={3}>
+            <Flex>
+              <Flex.Item flex="grow">
+                <Flex direction="column" gap={1}>
+                  <Text size="100">Ideas for you</Text>
+                  <Box display="flex">
+                    <Text ref={anchorRef} inline weight="bold">
+                      <Box marginEnd={2}>Small tattoos</Box>
+                    </Text>
+                  </Box>
+                </Flex>
+              </Flex.Item>
+              <Flex.Item flex="none">
+                <Icon
+                  accessibilityLabel=""
+                  icon="arrow-forward"
+                />
+              </Flex.Item>
+            </Flex>
+            <Box width="100%" aria-hidden>
+              <Mask rounding={1} wash>
+                <Image
+                  alt=""
+                  color="rgb(231, 186, 176)"
+                  loading="lazy"
+                  naturalHeight={181}
+                  naturalWidth={698}
+                  src="https://i.ibb.co/DWJFWkV/Screenshot-2023-02-24-at-2-27-59-PM.png"
+                />
+              </Mask>
+            </Box>
+          </Flex>
+        </Box>
+      </TapArea>
+      {open && (
+        <PopoverEducational
+          id="popover-do-educate"
+          idealDirection="right"
+          anchor={anchorRef.current}
+          onDismiss={() => {}}
+          message="Explore your recent searches in more details"
+        />
+      )}
+    </React.Fragment>
+)}`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection name={generatedDocGen?.displayName}>
+        <MainSection.Subsection
+          title="Keyboard navigation"
+          description={`
+PopoverEducational doesn't behave like regular popovers where they are open/closed upon user interaction, p.e. Tooltip, Dropdown, or ComboBox. PopoverEducational visibility is not directly controlled by the user; instead, its visibility is defined as part of a broader user experience and the user interaction engagement with this experience.
```

Comment:
typo: "_i_. e."

Reply:
p.e. is very me... cant help it

Reply:
DONE

Reply:
Oh, is "p.e." a thing people use? I don't think I've come across it before.

Also technically this should be "e.g.", not "i.e."…though not what they actually stand for, I remember usage like this:
`"i.e."` - "in other words"
`"e.g."` - "example(s) given"